inji-web/src/modals/DataShareExpiryModal.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {useState} from "react";
+import {ModalWrapper} from "./ModalWrapper";
+import {DataShareHeader} from "../components/DataShare/DataShareHeader";
+import {DataShareFooter} from "../components/DataShare/DataShareFooter";
+import {DataShareContent} from "../components/DataShare/DataShareContent";
+import {CustomExpiryModal} from "./CustomExpiryModal";
+import {useTranslation} from "react-i18next";
+
+export const DataShareExpiryModal: React.FC<DataShareExpiryModalType> = (props) => {
+
+    const [custom, setCustom] = useState<boolean>(false);
```

Comment:
what is this `custom` ?

Reply:
renamed it to "setIsCustomExpiryInTimesModalOpen"